en/frames_plane/reptile_dragon_2.md

```diff
@@ -0,0 +1,391 @@
+
+1. In the PX4 board config tool, navigate to the `drivers` submenu, then scroll down to highlight `rc_input`.
+2. Use the enter key to remove the `*` from `rc_input` checkbox.
+3. Scroll to highlight the `RC` submenu, then press enter to open it.
+4. Scroll to highlight `crsf_rc` and press enter to enable it.
+5. Save and exit the PX4 board config tool.
+
+### `msp_osd` Module
+
+The `msp_osd` module steams MSP telemetry to a selected serial port.
+The Caddx Vista Air Unit supports listening to MSP telemetry and will show the received telemetry values in its OSD (on screen display).
+
+1. In the PX4 board config tool, navigate to the `drivers` submenu, then scroll down to highlight `OSD`.
+2. Use the enter key to open the `OSD` submenu
+3. Scroll down to highlight `msp_osd` and press enter to enable it
+
+### Parameter Config
+
+- Use the [MSP_OSD_CONFIG](../advanced_config/parameter_reference.md#MSP_OSD_CONFIG) param to select the serial port which is connected to the Caddx Vista (in this build, `/dev/ttyS7`).
+- Use the [RC_CRSF_PRT_CFG](../advanced_config/parameter_reference.md#RC_CRSF_PRT_CFG) param to select the serial port which is connected to the ELRS RX (in this build, `telem 1`).
+
+## Preflight & First Flight
+
+### Radio Setup
+
+For a first flight, I recommend to have at least `manual`, `acro`, and `position` flight modes accessible.
+Additionally, I also attach an aux switch to the fixed wing autotune module so that the airframe can be easily autotuned during the first flight.
+
+The channel mappings for this build are included in the params file linked at the bottom of this page.
+I chose to map the channels in order throttle, roll, pitch, yaw, (blank), and flight mode
+
+:::note
+ExpressLRS requires AUX1 as an "arming channel".
+This arming channel is separate from PX4's arming mechanism and is used to tell the ELRS TX that is can switch into high transmit power.
+
+In the PX4 channel mappings, I simply skip over this channel.
+On my transmitter, this channel is set to always be "high", so ELRS is always armed.
+:::
+
+### Motor Setup & Prop Installation
+
+The RD2 kit comes with clockwise and counter clockwise propellers for counter rotating motors.
+With counter rotating props, the airplane can be set up such that it has no [critical motors](https://en.wikipedia.org/wiki/Critical_engine).
+
+With no critical motors, controllability will be maximized if a motor fails.
+The motor direction should be set such that props should turn towards the fuselage on top of the plane.
+In other words, if you look at the left motor with the airplane facing away from you, it should spin clockwise while the right motor should spin counter clockwise.
+
+With the propellers removed, power the airplane up and use the [Actuator](../config/actuators.md) test in QGC to spin up the motors.
+If the left or right motor does not spin in the correct direction, swap two of its ESC leads and check it again.

```

Comment:
Can't you change the direction in the actuators page?

Reply:
The ESCs have a throttle range of 1000 ... 2000, so they cannot be reversed in software